client/src/components/mapping/FhirMappingPanel/FhirResourceTree/index.tsx

```diff
@@ -60,6 +61,16 @@ const FhirResourceTree = ({ onClickCallback }: Props) => {
   const attributesForResource: { [k: string]: IAttribute } = useSelector(
     (state: IReduxStore) => state.resourceInputs.attributesMap
   );
+  const attributesWithInputs = Object.keys(attributesForResource).filter(
+    path => attributesForResource[path].inputGroups.length > 0
```

Comment:
may an inputGroup be empty ?

Reply:
Only if something bad happens... But I still fixed that